.gitignore

```diff
@@ -15,3 +15,6 @@
 # Ignore local environment variables which can contain environment secrets
 .env
 .envrc
+
+# Terraform files
+**/.terraform.lock.hcl
```

Comment:
Without the `**` it doesn't pick up the file if it's not in the root. I added this when a terraform lock file was accidentally committed

Reply:
`.terraform.lock.hcl` works without the `**`. i think the issue is that the .gitignore file changes didn't actually propagate to the various platform test repos since install-template.sh doesn't actually copy the .gitignore file over. so it looks like the `.terraform.lock.hcl` is missing from .gitignore in platform-test.